static/app/components/stream/group.tsx

```diff
@@ -496,7 +514,7 @@ function BaseGroupRow({
             <EventCountsWrapper>{groupCount}</EventCountsWrapper>
           )}
           {withColumns.includes('users') && issueTypeConfig.stats.enabled && (
-            <EventCountsWrapper>{groupUsersCount}</EventCountsWrapper>
+            <EventUsersCountsWrapper>{groupUsersCount}</EventUsersCountsWrapper>
```

Comment:
The EventCountsWrapper has been given slightly less left margin to better align the event stats graph with the column header